snippets/getColonTimeFromDate.md

```diff
@@ -0,0 +1,11 @@
+### getColonTimeFromDate
+
+Formats hour, minute, and second time integers from Date into stringified colon representation. Formats hour integer to 12 hour clock and maintains digit lengths to match (HH:MM:SS).
+
+```js
+const getColonTimeFromDate = date => date.toTimeString().slice(0, 8)
```

Comment:
Hey @Emgo-Dev 👋 😄 

Also I believe going about the following makes things much easier

Changing the method name to `digitalClock24`

```js
const digitalClock24 = _ => new Date().toTimeString().slice(0, 8);
```

Which can now be invoked by doing the following

```js
digitalClock24() // "13:07:00"

```

Reply:
Yeah that makes a lot more sense. I think my original terminology was a bit more accurate too. I was following the changes made to my toOrdinal() submission. The file name was changed to get ordinal suffix from integer. So I just followed that.